chrome/skin/style-osx.css

```diff
@@ -0,0 +1,7 @@
+@import 'style.css';
+
+#universal-search-recommendation {
+  background-color: Highlight;
+  color: HighlightText;
+  background-image: linear-gradient(rgba(255,255,255,0.3), transparent);
+}
```

Comment:
This little bit in the diff:

![screen shot 2016-03-21 at 8 37 29 am](https://cloud.githubusercontent.com/assets/23885/13921999/2ad7fd70-ef40-11e5-8a03-9833007e6902.png)

Means that there's no newline at the end of the file. Would you mind adding that for all the CSS files you added?